util/Migrator/DbScripts/2021-03-04_03_OrganizationUser_Enlarge_Email_Column.sql

```diff
@@ -0,0 +1,371 @@
+-- Check email column is 50 characters long (100 bytes)
+IF COL_LENGTH('[dbo].[OrganizationUser]', 'Email') = 100
+BEGIN
+	ALTER TABLE [dbo].[OrganizationUser] 
+	ALTER COLUMN 
+		Email NVARCHAR(256) NOT NULL
+END
+GO
+
+-- Recreate procedure OrganizationUser_Create
+IF OBJECT_ID('[dbo].[OrganizationUser_Create]') IS NOT NULL
+BEGIN
+    DROP PROCEDURE [dbo].[OrganizationUser_Create]
+END
+GO
+
+CREATE PROCEDURE [dbo].[OrganizationUser_Create]
+    @Id UNIQUEIDENTIFIER,
+    @OrganizationId UNIQUEIDENTIFIER,
+    @UserId UNIQUEIDENTIFIER,
+    @Email NVARCHAR(256),
+    @Key VARCHAR(MAX),
+    @Status TINYINT,
+    @Type TINYINT,
+    @AccessAll BIT,
+    @ExternalId NVARCHAR(300),
+    @CreationDate DATETIME2(7),
+    @RevisionDate DATETIME2(7),
+    @Permissions NVARCHAR(MAX)
+AS
+BEGIN
+    SET NOCOUNT ON
+
+    INSERT INTO [dbo].[OrganizationUser]
+    (
+        [Id],
+        [OrganizationId],
+        [UserId],
+        [Email],
+        [Key],
+        [Status],
+        [Type],
+        [AccessAll],
+        [ExternalId],
+        [CreationDate],
+        [RevisionDate],
+        [Permissions]
+    )
+    VALUES
+    (
+        @Id,
+        @OrganizationId,
+        @UserId,
+        @Email,
+        @Key,
+        @Status,
+        @Type,
+        @AccessAll,
+        @ExternalId,
+        @CreationDate,
+        @RevisionDate,
+        @Permissions
+    )
+END
+GO
+
+-- Recreate procedure OrganizationUser_Update
+IF OBJECT_ID('[dbo].[OrganizationUser_Update]') IS NOT NULL
+BEGIN
+    DROP PROCEDURE [dbo].[OrganizationUser_Update]
+END
+GO
+
+CREATE PROCEDURE [dbo].[OrganizationUser_Update]
+    @Id UNIQUEIDENTIFIER,
+    @OrganizationId UNIQUEIDENTIFIER,
+    @UserId UNIQUEIDENTIFIER,
+    @Email NVARCHAR(256),
+    @Key VARCHAR(MAX),
+    @Status TINYINT,
+    @Type TINYINT,
+    @AccessAll BIT,
+    @ExternalId NVARCHAR(300),
+    @CreationDate DATETIME2(7),
+    @RevisionDate DATETIME2(7),
+    @Permissions NVARCHAR(MAX)
+AS
+BEGIN
+    SET NOCOUNT ON
+
+    UPDATE
+        [dbo].[OrganizationUser]
+    SET
+        [OrganizationId] = @OrganizationId,
+        [UserId] = @UserId,
+        [Email] = @Email,
+        [Key] = @Key,
+        [Status] = @Status,
+        [Type] = @Type,
+        [AccessAll] = @AccessAll,
+        [ExternalId] = @ExternalId,
+        [CreationDate] = @CreationDate,
+        [RevisionDate] = @RevisionDate,
+        [Permissions] = @Permissions
+    WHERE
+        [Id] = @Id
+
+    EXEC [dbo].[User_BumpAccountRevisionDate] @UserId
+END
+GO
+
+-- Recreate procedure Organization_Search
+IF OBJECT_ID('[dbo].[Organization_Search]') IS NOT NULL
+BEGIN
+    DROP PROCEDURE [dbo].[Organization_Search]
+END
+GO
+
+CREATE PROCEDURE [dbo].[Organization_Search]
```

Comment:
Why repeat this again in another script? I mean it doesn't really "hurt" anything, but curious why it's repeated.

Reply:
Remove the whole CREATE PROC block.